modules/ROOT/pages/workday/workday-connector-config-topics.adoc

```diff
@@ -13,7 +13,7 @@ Configure the Global element for the connector.
 |Connection |The connection associated with the configuration.
 |Username |Enter the username to log in with.
 |Password |Enter the corresponding password.
-|Tenant Name | Enter the Workday Tenant ID, which usually has the suffix `pt_1` appended to it, such as `acme_pt1`. 
+|Tenant Name | Enter the Workday Tenant ID, which usually has the suffix `pt_1` appended to it, such as `acme_pt1`.
```

Comment:
The two suffixes do not match:  `pt_1` appended to it, such as `acme_pt1`.

Is it pt_1 or _pt1?